lms/templates/preview_menu.html

```diff
@@ -66,13 +86,61 @@
                     </p>
                 </div>
             % endif
-            % if microfrontend_link:
             <div style="flex-grow: 1; text-align: right;">
```

Comment:
@davidjoy Do you have any tricks on how to get these buttons to collapse better on smaller widths?
I'm happy to look for other examples in platform, just seeing if you have anything on hand ;)

Reply:
If you add `display: flex` to this div, it'll cause the buttons to at least stay side-by-side.  At very small widths (less than 460w on my screen) the insights button is partially off screen, but at least it doesn't take up more vertical space.  Hm, I also don't have the MFE link enabled, but even if it was, I think they'd all stay on a line.

Beyond that, I think you have two options that I can think of:

- You could vertically align the stuff in the toolbar - probably a non-starter because it'd take up so much space.
- You could change them to an icon-based buttons instead of text, or a smaller font size, at narrower resolutions so that they take up less room.  Or maybe a dropdown.  You'd probably have to do this via media queries in CSS.  Not that bad, but you'd have to play around a bit to get it right.

I think the latter is probably your best bet for making it "right" at small resolutions.  Whether staff/instructors are really going to go look at studio/insights on a mobile phone, though... not clear to me it's worth it, though it galls me to leave it not-quite-fitting. :)